blimp-live-widgets/example/public/index.html

```diff
@@ -5,6 +5,9 @@
     <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <meta name="theme-color" content="#000000">
 
+    <script src="https://apis.google.com/js/platform.js" async defer></script>
```

Comment:
I had to deal with something where I had to add a script tag on the Twitter component (maybe check it out?). I think this should be added to the widget / component itself this example will not be exported when imported by the web repo so if we need the script it should probs be there.